sql/init.sql

```diff
@@ -0,0 +1,59 @@
+CREATE DATABASE performee;
+
+CREATE TABLE IF NOT EXISTS public.plans
```

Comment:
I think it should be part of liquibase? This is part of business objects and should not be part of init script.